action.yml

```diff
@@ -25,6 +25,7 @@ inputs:
     default: "broad-jade-dev"
   k8_cluster:
     description: "Google kubernetes cluster for sdk ie:integration-master"
+    default: "integration-master"
```

Comment:
pull this out we should make a habit of setting this as later it wont be `"integration-master"`